src/INTERLAYER/pair_ilp_tmd.cpp

```diff
@@ -479,9 +481,63 @@ void PairILPTMD::calc_normal()
           for (m = 0; m < Nnei; m++) { dnormal[i][id][m][ip] = 0.0; }
         }
       }
+      // for hydrogen in water molecule
+      if (strcmp(elements[itype], "Hw") == 0) {
+        if(cont == 0) {
+          jH1 = atom->map(tag[i] - 1);
+          jH2 = atom->map(tag[i] - 2);
+          iH1 = map[type[jH1]];
+          iH2 = map[type[jH2]];
+          if (strcmp(elements[iH1], "Ow") == 0 ) {
+            vect[0][0] = x[jH1][0] - xtp;
+            vect[0][1] = x[jH1][1] - ytp;
+            vect[0][2] = x[jH1][2] - ztp;
+          } else if (strcmp(elements[iH2], "Ow") == 0 ) {
+            vect[0][0] = x[jH2][0] - xtp;
+            vect[0][1] = x[jH2][1] - ytp;
+            vect[0][2] = x[jH2][2] - ztp;
+          } else {
+            fprintf(screen, "jH1 jH2 = %d %d\n", jH1,jH2);
```

Comment:
This is effectively debug output thus please remove the fprintf() calls and only leave the error->one() call.

Reply:
Removed

Reply:
Thanks.